RobotLocomotion/drake
# Add ExternallyAppliedSpatialForceMultiplexer

Resolves #13139
Relates #16923

py plant:
- Remove unneeded qualifier for ExternallyAppliedSpatialForce
- test_propeller: Use body name rather than magic number

adder_test:
- Add check for discrete number of states
- Use std::make_unique instead of .reset()

\cc @amcastro-tri @jwnimmer-tri 


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/robotlocomotion/drake/18171)
